src/lib/__tests__/__cardFunctionsTests__/suitChar.test.js

```diff
@@ -0,0 +1,44 @@
+const { suitChar } = require("lib/cardFunctions");
+
+describe("suitChar function", () => {
+  test("should return symbol: <>&hearts;</>, string: hearts ", () => {
+    //GIVEN
+    const suitCharacter = "H";
+    //WHEN
+    const result = suitChar(suitCharacter);
+    //THEN
+    expect(result).toStrictEqual({ symbol: <>&hearts;</>, string: "hearts" });
```

Comment:
toEqual i think is enough and applies for all the cases in this PR

Reply:
Done.